MAUI/DataGrid/row-drag-and-drop.md

```diff
@@ -0,0 +1,402 @@
+---
+layout: post
+title: Row Drag and Drop in .NET MAUI DataGrid control | Syncfusion
+description: Learn here all about Row Drag and Drop support in Syncfusion MAUI DataGrid (SfDataGrid) control and more here.
+platform: MAUI
+control: SfDataGrid
+documentation: ug
+keywords : maui datagrid, maui grid, grid maui, maui gridview, grid in maui, .net maui datagrid, .net maui grid, .net grid maui, .net maui row drag and drop, maui row drag and drop
+---
+
+# Row Drag and Drop in MAUI DataGrid (SfDataGrid)
+
+The SfDataGrid allows you to drag and drop a row by setting the `SfDataGrid.AllowDraggingRow` property to `true`. A customizable row drag and drop template is displayed while a row is being dragged.  Based on the requirements, drag and drop operations can be handled by using the `SfDataGrid.QueryRowDragging` event. 
+
+To enable row drag and drop functionality, please follow the code example below:
+
+{% tabs %}
+{% highlight XAML %}
+<syncfusion:SfDataGrid x:Name="dataGrid"
+                       ItemsSource="{Binding OrderInfoCollection}"  
+                       AllowDraggingRow="True">
+ </syncfusion:SfDataGrid>
+{% endhighlight %}
+{% highlight c# %}
+dataGrid.AllowDraggingRow = true;
+{% endhighlight %}
+{% endtabs %}
+
+<img alt="DataGrid row drag and drop" src="Images\row-drag-and-drop\Dragging.gif" width="404"/>
+
+## Dragging scenarios
+
+The SfDataGrid supports drag and drop operations for both data rows and groups in the following scenarios:
+
+ * Records can be reordered to any position using auto-scrolling.
+ * Group positions can be reordered using drag and drop. However, no groups can be added within other groups.
+ * Data rows can be reordered within the same group or moved to other groups as well.
+
+N> Reordering changes are made only in the `SfDataGrid.View`, and not in the underlying data. Thus, the changes will be reverted when performing sorting, grouping, or any other operation refreshing the view. Reordering changes in the underlying data can be achieved by handling a `QueryRowDragging` event in the sample side as explained below in [Reordering underlying data](# Reordering underlying data ).
+
+N> Now you can drag and drop the grid rows using your mouse in the UWP (Desktop) platform.
+
+## Row drag-and-drop template
+
+The data grid allows you to load desired content when performing row drag-and-drop operations using the `SfDataGrid.RowDragDropTemplate`. The template can be defined either in code or XAML.  
+
+## Default template
+
+The default template will be used if no other template is explicitly assigned for row drag-and-drop operations.
```

Comment:
The default template will be used for row drag-and-drop operations unless another template is explicitly assigned.

Reply:
Updated